account_payment_order/models/account_payment_order.py

```diff
@@ -327,13 +322,7 @@ def draft2open(self):
                             pdate=requested_date,
                         )
                     )
-                # Write requested_date on 'date' field of payment line
-                # norecompute is for avoiding a chained recomputation
-                # payment_line_ids.date
-                # > payment_line_ids.amount_company_currency
-                # > total_company_currency
-                with self.env.norecompute():
-                    payline.date = requested_date
+                payline.date = requested_date
```

Comment:
@pedrobaeza I have removed this because in v17 it is obsolete and is not doing anything, but I am not sure how to check if it is a problem.